test/parallel/test-cluster-fork-windowsHide.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+
+if (cluster.isMaster) {
+  const msg = 'foobar';
+
+  cluster.setupMaster({
+    silent: true,
+    windowsHide: true
+  });
+
+  const worker = cluster.fork();
+  const cstdin = worker.process.stdin;
+  const cstdout = worker.process.stdout;
+  let response = '';
+
+  worker.on('exit', common.mustCall((code, signal) => {
+    assert.strictEqual(code, 0);
+    assert.strictEqual(signal, null);
+  }));
+
+  cstdout.on('data', (data) => {
```

Comment:
Call `cstdout.setEncoding('utf8')`, then you don't have to `.toString('utf8')` below.

Reply:
Yeah, you are right, `cstdout.setEncoding('utf8') ` make more sense here.  Just deal too much binary data these days :D .